### ISSUE TITLE
Why  BL-PF and other Baseline don't use model.eval()?

### ISSUE THREAD

**Issue by Suetsumu** [IBM/reprogrammble-FL#1]
When you use DP, you should only train model with one sample. You set model.eval() in MR_Centralized.py https://github.com/IBM/reprogrammble-FL/blob/e031dbbb4345011d7196019bd6e43fdb30c07325/MR_Centralized.py#L152
 but you don't apply this configuration in your baseline https://github.com/IBM/reprogrammble-FL/blob/e031dbbb4345011d7196019bd6e43fdb30c07325/BL_PF_Cent.py#L60 When you train BL-PF, BN will update running_mean and running_var automatically. When you train your model with only one sample, BN will show a very poor performance, so BL-PF perform badly[1]. When I apply this setting on  BL-PF, I get a great acc further better than your report. 

[1]Wu Y, He K. Group normalization[C]//Proceedings of the European conference on computer vision (ECCV). 2018: 3-19.
